app/inject.js

```diff
@@ -68,6 +68,25 @@ chrome.extension.onMessage.addListener(function(msg, sender, sendResponse) {
         args.cookies = document.cookie;
     }
 
+    if (msg.tradesNotification) {
+        var tradesBtn = $('#menu>a').eq(1);
```

Comment:
A better and much safer way would be selecting link by the URL, otherwise if some other extension added a menu link or the site itself adds another link then it will break. In background.js we have this:
```
    var trades = data.find('a[href$="mytrades"]:first');
    var offers = data.find('a[href$="myoffers"]:first');
```